streams/src/main/java/org/apache/kafka/streams/state/internals/TimestampedKeyValueStoreBuilder.java

```diff
@@ -78,7 +78,7 @@ public TimestampedKeyValueStoreBuilder(final KeyValueBytesStoreSupplier storeSup
         if (!enableCaching) {
             return inner;
         }
-        return new CachingKeyValueStore(inner);
+        return new CachingKeyValueStore(inner, true);
```

Comment:
We need to know what format of values the cache stores, whether they are timestamped or not